database/rrddim.c

```diff
@@ -211,7 +211,7 @@ void rrdcalc_link_to_rrddim(RRDDIM *rd, RRDSET *st, RRDHOST *host) {
     }
 #ifdef ENABLE_ACLK
     if (netdata_cloud_setting)
-        aclk_update_chart(st->rrdhost, st->id, ACLK_CMD_CHART);
+        rrdset_flag_set(st, RRDSET_FLAG_ACLK);
```

Comment:
all the places like this can always set the flag (regardless of netdata_cloud_setting). This way we avoid branching

Reply:
True yes, the flag is checked again in `aclk_update_chart`